src/hotspot/cpu/x86/stubGenerator_x86_64_tanh.cpp

```diff
@@ -66,11 +66,11 @@
 //
 // For |x|<2^{-64}:  x is returned
 //
-// For |x|>=2^32: return +/-1
+// For |x|>=20: return +/-1
 //
 // Special cases:
 //  tanh(NaN) = quiet NaN, and raise invalid exception
-//  tanh(INF) = that INF
+//  tanh(+/-INF) = +/-1
```

Comment:
Thanks for fixing this. Yudi Zheng (@mur47x111) posted a comment about this on my PR after it was integrated.